pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -348,6 +349,7 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 		}
 	}
 
+	UpdateIpsecTelemetry(&newOperConfig.Spec)
```

Comment:
I think it would be good to introduce new field in `ReconcileOperConfig` struct to store IPsec mode and update ipsec metric when it changes to a new value.